.github/workflows/pr-test.yml

```diff
@@ -39,9 +42,19 @@ jobs:
           fail_ci_if_error: true
           verbose: true
           flags: unit
+      
+      - name: Archive unit test report
+        uses: actions/upload-artifact@v1
+        with:
+          name: unit-test
+          path: /unit-test-reports
+        if: ${{ always() }}
 
       - name: Integration test
-        run: ./gradlew jacocoIntegrationTestReport
+        run: ./gradlew jacocoIntegrationTestReport copyAllReports --output-dir=/integration-test-reports
+
+      - name: Copy integration test reports
+        run: ./gradlew copyAllReports --output-dir=/tmp/integration-test-reports
```

Comment:
this output dir doesn't match the upload directory on line 71

Reply:
fixed this and everything is working now: https://github.com/Fininity-ai/Attributes-test/runs/1586845665